packages/teams-js/src/public/call.ts

```diff
@@ -6,14 +6,21 @@ import { errorNotSupportedOnPlatform, FrameContexts } from './constants';
 import { runtime } from './runtime';
 
```

Comment:
Description for the overall namespace?

Reply:
Sure, will add description for the namespace. 
Trevor -  TypeDoc has no check for namespace. Also, the tag `[@namespace](https://typedoc.org/tags/namespace/)` can be used to tell TypeDoc to convert a variable as a namespace.